lib/cyclonedx/cocoapods/spdx_licenses.rb

```diff
@@ -0,0 +1,497 @@
+# List of license ids as defined in the SPDX schema, v1.0-3.10
```

Comment:
Is it possible to use an existing SPDX license list rather than having the list in code? CycloneDX generates license lists that are included with other implementations. For example https://github.com/CycloneDX/cyclonedx-node-module/blob/master/spdx-licenses.json.

Reply:
🤔 I generated that list directly from https://cyclonedx.org/schema/spdx so what would we gain using an officially generated license list? Anyway, where are those lists published?

Reply:
When CycloneDX updates its official license list, it also generates a JSON of all valid license IDs. This list is generate4d automatically and will eventually be automatically submitted to each of the official CycloneDX implementations. Having all implementations use the same license json file will simplify the automation.

Reply:
Fair enough, fixed in b28de128947f302745b9633e16bb3b8a7dc349ed